example/ndpiReader.c

```diff
@@ -2427,6 +2427,10 @@ static void setupDetection(u_int16_t thread_id, pcap_t * pcap_handle) {
   if(_maliciousSHA1Path)
     ndpi_load_malicious_sha1_file(ndpi_thread_info[thread_id].workflow->ndpi_struct, _maliciousSHA1Path);
 
+  /* Enable/disable/configure LRU caches here */
+  ndpi_set_lru_cache_size(ndpi_thread_info[thread_id].workflow->ndpi_struct,
+			  NDPI_LRUCACHE_BITTORRENT, 32768);
```

Comment:
This statement is not required and just here for documentation purposes, am I right?